models/user.js

```diff
@@ -1,18 +1,10 @@
 const { Schema, model } = require('mongoose')
-const {
-  roles: { STUDENT, MENTOR, ADMIN }
```

Comment:
delete roles from file

Reply:
it's not my PR, I do not have it in mine

Reply:
it's not my PR either, I do not have it in mine too, maybe you have

Reply:
![image](https://user-images.githubusercontent.com/79531224/182551213-08ab7fd5-bbec-47e6-8d7c-602e28e13139.png)